spl-balance/src/pb.rs

```diff
@@ -0,0 +1,9 @@
+#[rustfmt::skip]
+#[path = "../target/pb/messari.sol_token.v1.rs"]
+pub(in crate::pb) mod sol_token_v1;
```

Comment:
Just noting this here for the future but what we should do here is filter out all protos which are from substream-helpers in the build.rs script and not build bindings for them. Then we can just import the bindings from the substreams-helper crate instead which will avoid the unnecessary duplication of code here